docs/GettingStartedDocs/Contributors/NonAccMachineGettingStarted.md

```diff
@@ -0,0 +1,228 @@
+# Configure OE SDK on non ACC VM
+
+## Disclaimer
+This document is to provide a viable solution to enable openenclave SGX DCAP remote attestation to run on non Azure Confidential Computing(ACC) machines. It relies on several Intel components and services which are subject to Intel's changes.
```

Comment:
Is this is specific to SGX-only, then please put SGX in the filename and title on line 1

Reply:
Thanks for your comments. Changed the filename and the title accordingly.